app/ui/lib/Table.tsx

```diff
@@ -115,7 +115,7 @@ Table.Cell = ({ height = 'small', className, children, ...props }: TableCellProp
  * Used _outside_ of the `Table`, this element wraps buttons that sit on top
  * of the table.
  */
-export const TableActions = classed.div`-mt-11 mb-3 flex justify-end gap-2`
+export const TableActions = classed.div`-mt-6 mb-3 flex justify-end gap-2`
```

Comment:
I don't like it lol. but it'll look a lot better when the table actions row is more filled out

Reply:
Was weird to have it shuffle on different pages – instances and projects.